### DISCUSSION TITLE
Get error if one crud page has two datetime_picker

### DISCUSSION THREAD

**Discussion by amigolj** [Laravel-Backpack/community-forum#907] [Q&A (Help)]
Hi, when I add second datetime_picker field  to the page with crud, both field don't work and I get error in the browser console but when I deleted second field with datetime_picker, it's works correctly
![Screenshot_4](https://github.com/Laravel-Backpack/community-forum/assets/10392966/38e70b72-e748-430b-b67f-eba531efc16a)
![Screenshot_1](https://github.com/Laravel-Backpack/community-forum/assets/10392966/b4a589ac-aec2-423a-8b48-7857335fc7f7)
UPDATE
I'm not using the standard display format
![image](https://github.com/Laravel-Backpack/community-forum/assets/10392966/5d051c8f-c04a-456a-8fbd-d7cfe06dd701)

**Comment by karandatwani92** [comment]
Hey @amigolj 

I just tried adding two `datetime_picker` in the same CRUD with the following code; it works for me.
```php
$this->crud->addFields([
   'name'  => 'datetime',
   'type'  => 'datetime_picker',
   // optional:
   'datetime_picker_options' => [
      'format'   => 'DD/MM/YYYY HH:mm',
      'language' => 'en',
   ],
   'wrapperAttributes' => ['class' => 'form-group col-md-6'],
],
[
   'name'  => 'datetime_picker',
   'type'  => 'datetime_picker',
   // optional:
   'datetime_picker_options' => [
      'format'   => 'DD/MM/YYYY HH:mm',
      'language' => 'en',
   ],
   'wrapperAttributes' => ['class' => 'form-group col-md-6'],
]);
```
![Screenshot 2024-03-30 at 11 26 34 AM](https://github.com/Laravel-Backpack/community-forum/assets/8214221/5cabdd86-55cf-45f3-a223-4e4655b2f881)

You may
- Remove custom `datetime_picker.blade.php` if you have overridden the field.
- Try `composer update` and check again with the latest backpack version.

**Reply by amigolj** [reply]
I apologize to you for not giving enough information, I'm not using the standard display format
![image](https://github.com/Laravel-Backpack/community-forum/assets/10392966/5d051c8f-c04a-456a-8fbd-d7cfe06dd701)

**Reply by amigolj** [reply]
I checked, with standart format, two datetime pickers works. I am looking for a convenient time selection field, the standard one is not very convenient as you have to select PM AM non-English speaking people for them may have a problem understanding what to select
